core/parameters/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg_attr(enable_const_type_id, feature(const_type_id))]
```

Comment:
Why do we need this new line? I could see no other change in this PR that would require this

Reply:
I need to add this to every crate structs of which I want to guard. This is already in place for near-primitives crate.

The way how I made it work is creating `ProtocolSchemaInfo` for every struct and registering them using `inventory`. To navigate across structs, I also want to record `TypeId`. But `ProtocolSchemaInfo` must be static, and then enabling `const_type_id` becomes a requirement.

Example of resulting struct:
```
pub static WeightedIndex_INFO: near_structs_checker_lib::ProtocolSchemaInfo = near_structs_checker_lib::ProtocolSchemaInfo::Struct {
        name: "WeightedIndex",
        type_id: std::any::TypeId::of::<WeightedIndex>(),
        fields: &[...]
}
```